osu.Game/Overlays/OverlayHeader.cs

```diff
@@ -16,24 +18,19 @@ public abstract class OverlayHeader : Container
         private readonly Box titleBackground;
         private readonly Box controlBackground;
         private readonly Container background;
-
-        protected Color4 TitleBackgroundColour
-        {
-            set => titleBackground.Colour = value;
-        }
-
-        protected Color4 ControlBackgroundColour
-        {
-            set => controlBackground.Colour = value;
-        }
+        private readonly ScreenTitle title;
 
         protected float BackgroundHeight
         {
             set => background.Height = value;
         }
 
-        protected OverlayHeader()
+        protected OverlayColourScheme ColourScheme { get; private set; }
```

Comment:
Why? We don't want to allow to change this elsewhere. However we can use a readonly field, but not sure if we want a field here.

Reply:
well, fyi, making a property get-only means it can only be set in-line statically `private Type A { get; } = 1;` or at constructor. (which is what you're doing), not allowing to be set publicly.

Reply:
Ah that's what you mean, sure then